bin/setup

```diff
@@ -1,29 +1,40 @@
 #!/usr/bin/env ruby
-require 'pathname'
+require "pathname"
+require "fileutils"
+
+include FileUtils
+
+ENV["RAILS_ENV"]               ||= ENV["PASSENGER_APP_ENV"]
+ENV["RAILS_RELATIVE_URL_ROOT"] ||= ENV["PASSENGER_BASE_URI"]
```

Comment:
Actually, we should use a sensible default for RAILS_ENV. If PASSENGER_APP_ENV is not set, we need to set the default. Either default to production or development. Rails convention is development, but I think we should default to production. Or forget defaults, and exit with error if `PASSENGER_APP_ENV` or `RAILS_ENV` is not specified. But don't throw an error about `RAILS_RELATIVE_URL_ROOT` since that is normally specified in the dotenv file.